src/RoatpCourseManagement/SFA.DAS.RoatpCourseManagement/Application/UkrlpData/UkrlpProfile.cs

```diff
@@ -0,0 +1,89 @@
+﻿
+
+using AutoMapper;
+
+namespace SFA.DAS.RoatpCourseManagement.Application.UkrlpData
+{
+    public class UkrlpProviderDetailsProfile : Profile
+    {
+        public UkrlpProviderDetailsProfile()
+        {
+            CreateMap<MatchingProviderRecords, ProviderDetails>()
```

Comment:
I am not in favour of using automapper. We have used implicit operator elsewhere and I think we should be consistent in our approach.

Reply:
Ok, will do implicits for mapping, and will remove automapper